pkgs/servers/http/apache-httpd/2.2.nix

```diff
@@ -23,6 +23,7 @@ stdenv.mkDerivation rec {
   # FIXME: -dev depends on -doc
   outputs = [ "dev" "out" "doc" ];
   setOutputFlags = false; # it would move $out/modules, etc.
+  stripDebugList= [ "bin" "lib" "modules" ]; # also strip modules.
```

Comment:
No, it does not change anything to apache 2.2, it simply unifies the syntax with 2.4.